op-deployer/pkg/deployer/inspect/l1.go

```diff
@@ -21,39 +20,6 @@ type L1Contracts struct {
 	ImplementationsDeployment ImplementationsDeployment `json:"implementationsDeployment"`
 }
 
-const (
-	SuperchainBundle      = "superchain"
-	ImplementationsBundle = "implementations"
-	OpChainBundle         = "opchain"
-)
-
-var ContractBundles = []string{
-	SuperchainBundle,
-	ImplementationsBundle,
-	OpChainBundle,
-}
-
-func (l L1Contracts) GetContractAddress(name string, bundleName string) (common.Address, error) {
-	var bundle interface{}
-	switch bundleName {
-	case SuperchainBundle:
-		bundle = l.SuperchainDeployment
-	case ImplementationsBundle:
-		bundle = l.ImplementationsDeployment
-	case OpChainBundle:
-		bundle = l.OpChainDeployment
-	default:
-		return common.Address{}, fmt.Errorf("invalid contract bundle type: %s", bundleName)
-	}
-
-	field := reflect.ValueOf(bundle).FieldByName(name)
-	if !field.IsValid() {
-		return common.Address{}, fmt.Errorf("contract %s not found in %s bundle", name, bundleName)
-	}
-
-	return field.Interface().(common.Address), nil
-}
```

Comment:
This was an unused function

Reply:
We thank it for its contributions and wish it luck in its career